File: src/main/java/com/dorpeled/pterodactylcommandsmod/commands/HelpCommand.java
package com.dorpeled.pterodactylcommandsmod.commands;

import com.dorpeled.pterodactylcommandsmod.util.Constants;
import com.mojang.brigadier.CommandDispatcher;
import com.mojang.brigadier.context.CommandContext;
import net.minecraft.commands.CommandSourceStack;
import net.minecraft.commands.Commands;
import net.minecraft.network.chat.Component;

public class HelpCommand {
    public static void register(CommandDispatcher<CommandSourceStack> dispatcher) {
        dispatcher.register(Commands.literal("pterodactyl")
                .then(Commands.literal("help")
                        .executes(HelpCommand::executeHelp))
        );
    }

    private static int executeHelp(CommandContext<CommandSourceStack> context) {
        CommandSourceStack source = context.getSource();
        source.sendSuccess(() -> Component.literal(Constants.HELP_MESSAGE), false);
        return 1;
    }
}

File: src/main/kotlin/com/dorpeled/pterodactylcommandsmod/commands/HelpCommand.kt
package com.dorpeled.pterodactylcommandsmod.commands

import com.dorpeled.pterodactylcommandsmod.util.Constants
import com.mojang.brigadier.CommandDispatcher
import com.mojang.brigadier.context.CommandContext
import net.minecraft.commands.CommandSourceStack
import net.minecraft.commands.Commands
import net.minecraft.network.chat.Component

object HelpCommand {
    fun register(dispatcher: CommandDispatcher<CommandSourceStack?>?) {
        dispatcher?.register(
            Commands.literal("pterodactyl")
                .then(
                    Commands.literal("help")
                        .executes(HelpCommand::executeHelp)
                )
        )
    }

    private fun executeHelp(context: CommandContext<CommandSourceStack?>?): Int {
        context?.source?.sendSuccess({ Component.literal(Constants.HELP_MESSAGE) }, false)
        return 1
    }
}
